src/pages/ValidateLoginPage/index.js

```diff
@@ -58,7 +58,7 @@ class ValidateLoginPage extends Component {
             if (lodashGet(this.props, 'session.authToken')) {
                 // If already signed in, do not show the validate code if not on web,
                 // because we don't want to block the user with the interstitial page.
-                Navigation.goBack(false);
+                Navigation.goBack();
```

Comment:
This `false` was here for no reason also so I removed it.

Reply:
It's added for a reason, to unset the fallback and call `navigationRef.current.goBack()` instead of `navigate(fallbackRoute, 'UP')`

Reply:
But it doesn't work that way, the first argument is the route to go back to: https://github.com/Expensify/App/blob/6183609c577f2310928bb4e8ee23b52f6dd28e00/src/libs/Navigation/Navigation.js#L88, or am I missing something?

Reply:
Yes, but the fallback is not necessary. In this line we used to disable the fallback so the second condition won't be true (not to call `navigate(fallbackRoute, 'UP')`) and continue with the third option (`navigationRef.current.goBack()`).

Reply:
Ok, I get it now, thanks!